app/models/npq_application.rb

```diff
@@ -1,6 +1,21 @@
 # frozen_string_literal: true
 
 class NPQApplication < ApplicationRecord
+  VALID_FUNDING_ELIGIBILITY_STATUS_CODES = %w[
```

Comment:
I moved this constant from service as its better suited in the model, already discussed with @cwrw